util/graph.js

```diff
@@ -0,0 +1,77 @@
+/** Class that signifies a graph */
+class Graph {
+    /**
+      * The underlying graph object. This is an object with nodes and edges.
+      * The class contains an adjacency list property that represents the edges
+     * between each of the nodes. Each edge can have attributes such as weight (which 
+     * by default is 10) and whether or not the edge is directed.
+     * 
+      */
+    adjacencyList;
```

Comment:
https://github.com/tair/mb4-service/pull/58/files#r1170650269
"Can you rename to "graph"?"